Makefile

```diff
@@ -42,7 +42,7 @@ endif
 LDFLAGS := $(LDFLAGS) -X "main.MakeVersion=$(MAKE_VERSION)" -X "main.Version=$(GITEA_VERSION)" -X "main.Tags=$(TAGS)"
 
 PACKAGES ?= $(filter-out code.gitea.io/gitea/integrations/migration-test,$(filter-out code.gitea.io/gitea/integrations,$(shell GO111MODULE=on $(GO) list -mod=vendor ./... | grep -v /vendor/)))
-SOURCES ?= $(shell find . -name "*.go" -type f)
+SOURCES ?= $(shell find . -name "*.go" -type f) $(BINDATA)
```

Comment:
I think this is probably incorrect.

We only need to run generate (at least at present) if TAGS contains bindata. 

(I think there are a few places where we could do more generation but I don't think we have any plans to do that at present.)